pkg/env/env.go

```diff
@@ -133,10 +134,17 @@ func (v *ViperEnv) Unmarshal(value interface{}) error {
 			return err
 		}
 	}
-
 	return v.Viper.Unmarshal(value)
+
 }
 
+func (v *ViperEnv) parseJson() {
+	versions := v.Viper.GetStringMapStringSlice("agents.versions")
```

Comment:
is this the right way to do it? seems weird to have code for a specific config property in here.